--- a/Makefile.orig	2017-07-06 11:54:48.555208626 +0000
+++ a/Makefile	2017-07-06 11:55:28.241026456 +0000
@@ -15,17 +15,14 @@
 SHELL=/bin/sh
 
 # To assist in cross-compiling
-CC=gcc
-AR=ar
-RANLIB=ranlib
+CC=${CHOST}gcc
+AR=${CHOST}ar
+RANLIB=${CHOST}ranlib
 LDFLAGS=
 
 BIGFILES=-D_FILE_OFFSET_BITS=64
 CFLAGS=-Wall -Winline -O2 -g $(BIGFILES)
 
-# Where you want it installed when you do 'make install'
-PREFIX=/usr/local
-
 
 OBJS= blocksort.o  \
       huffman.o    \
--- a/bzip2.c.orig	2017-07-06 11:58:32.487823343 +0000
+++ a/bzip2.c	2017-07-06 11:59:25.783921649 +0000
@@ -128,7 +128,7 @@
 #if BZ_LCCWIN32
 #   include <io.h>
 #   include <fcntl.h>
-#   include <sys\stat.h>
+#   include <sys/stat.h>
 
 #   define NORETURN       /**/
 #   define PATH_SEP       '\\'
