diff --git frei0r-plugins-1.6.0-orig/src/Makefile.am frei0r-plugins-1.6.0/src/Makefile.am
index e9c2ddc..958fbcd 100644
--- frei0r-plugins-1.6.0-orig/src/Makefile.am
+++ frei0r-plugins-1.6.0/src/Makefile.am
@@ -331,16 +331,16 @@ plugindir = @libdir@/frei0r-1
 
 
 install-pluginLTLIBRARIES: $(plugin_LTLIBRARIES)
-	mkdir -p $(DESTDIR)/$(plugindir)
+	mkdir -p $(DESTDIR)$(plugindir)
 	list='$(plugin_LTLIBRARIES)'; \
 	for file in $$list; do \
-	  sofile=`basename $$file .la`.so; \
-	  $(INSTALL_PROGRAM) .libs/$$sofile $(DESTDIR)/$(plugindir); \
+	  sofile=`basename $$file .la`.a; \
+	  $(INSTALL_PROGRAM) .libs/$$sofile $(DESTDIR)$(plugindir); \
 	done
 
 uninstall-pluginLTLIBRARIES:
 	list='$(plugin_LTLIBRARIES)'; \
 	for file in $$list; do \
-	  sofile=`basename $$file .la`.so; \
-	  rm -f $(DESTDIR)/$(plugindir)/$$sofile; \
+	  sofile=`basename $$file .la`.a; \
+	  rm -f $(DESTDIR)$(plugindir)/$$sofile; \
 	done
