diff -ur mpg123-1.24.0-orig/libmpg123.pc.in mpg123-1.24.0/libmpg123.pc.in
--- mpg123-1.24.0-orig/libmpg123.pc.in	2017-02-27 15:30:32.000000000 +0800
+++ mpg123-1.24.0/libmpg123.pc.in	2017-05-25 19:17:07.521585100 +0800
@@ -7,5 +7,6 @@
 Description: An optimised MPEG Audio decoder
 Requires: 
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lmpg123 
+Libs: -L${libdir} -lmpg123
+Libs.private: -lshlwapi  
 Cflags: -I${includedir} 
