--- rtmpdump/Makefile.orig	2014-03-02 18:20:23.000000000 +0000
+++ rtmpdump/Makefile	2014-03-12 09:12:10.896078800 +0000
@@ -1,16 +1,16 @@
 VERSION=v2.4
 
-prefix=/usr/local
+#prefix=/usr/local
 
 CC=$(CROSS_COMPILE)gcc
 LD=$(CROSS_COMPILE)ld
 
-SYS=posix
-#SYS=mingw
+#SYS=posix
+SYS=mingw
 
-CRYPTO=OPENSSL
+#CRYPTO=OPENSSL
 #CRYPTO=POLARSSL
-#CRYPTO=GNUTLS
+CRYPTO=GNUTLS
 LIBZ=-lz
 LIB_GNUTLS=-lgnutls -lhogweed -lnettle -lgmp $(LIBZ)
 LIB_OPENSSL=-lssl -lcrypto $(LIBZ)
--- rtmpdump/librtmp/Makefile.orig	2014-03-02 18:20:23.000000000 +0000
+++ rtmpdump/librtmp/Makefile	2014-03-12 09:13:48.314949400 +0000
@@ -1,11 +1,11 @@
 VERSION=v2.4
 
-prefix=/usr/local
+#prefix=/usr/local
 
 incdir=$(prefix)/include/librtmp
 bindir=$(prefix)/bin
 libdir=$(prefix)/lib
-mandir=$(prefix)/man
+mandir=$(prefix)/share/man
 BINDIR=$(DESTDIR)$(bindir)
 INCDIR=$(DESTDIR)$(incdir)
 LIBDIR=$(DESTDIR)$(libdir)
@@ -15,9 +15,9 @@
 LD=$(CROSS_COMPILE)ld
 AR=$(CROSS_COMPILE)ar
 
-SYS=posix
-CRYPTO=OPENSSL
-#CRYPTO=GNUTLS
+SYS=mingw
+#CRYPTO=OPENSSL
+CRYPTO=GNUTLS
 DEF_POLARSSL=-DUSE_POLARSSL
 DEF_OPENSSL=-DUSE_OPENSSL
 DEF_GNUTLS=-DUSE_GNUTLS
--- rtmpdump/Makefile.orig	2014-03-12 09:38:43.190274500 +0000
+++ rtmpdump/Makefile	2014-03-12 09:40:10.906913100 +0000
@@ -26,7 +26,7 @@
 
 bindir=$(prefix)/bin
 sbindir=$(prefix)/sbin
-mandir=$(prefix)/man
+mandir=$(prefix)/share/man
 
 BINDIR=$(DESTDIR)$(bindir)
 SBINDIR=$(DESTDIR)$(sbindir)
--- rtmpdump/librtmp/Makefile.orig	2014-03-12 09:15:13.992329000 +0000
+++ rtmpdump/librtmp/Makefile	2014-03-12 09:15:47.418573600 +0000
@@ -65,7 +65,7 @@
 INSTALL_IMPLIB=$(INSTALL_IMPLIB_$(SYS))
 
 SHARED=yes
-SODEF_yes=-fPIC
+SODEF_yes=
 SOLIB_yes=librtmp$(SO_EXT)
 SOINST_yes=install_so
 SO_DEF=$(SODEF_$(SHARED))
--- rtmpdump/thread.c.orig	2014-03-02 18:20:23.000000000 +0000
+++ rtmpdump/thread.c	2014-03-12 09:20:02.337444200 +0000
@@ -30,10 +30,14 @@
 ThreadCreate(thrfunc *routine, void *args)
 {
   HANDLE thd;
+  uintptr_t success;
 
-  thd = (HANDLE) _beginthread(routine, 0, args);
-  if (thd == -1L)
+  success = _beginthreadex(NULL, 0, routine, args, 0, (unsigned *)&thd);
+  if (!success)
+  {
     RTMP_LogPrintf("%s, _beginthread failed with %d\n", __FUNCTION__, errno);
+    return INVALID_HANDLE_VALUE;
+  }
 
   return thd;
 }
--- rtmpdump/thread.h.orig	2014-03-02 18:20:23.000000000 +0000
+++ rtmpdump/thread.h	2014-03-12 09:22:17.183567500 +0000
@@ -25,8 +25,8 @@
 #ifdef WIN32
 #include <windows.h>
 #include <process.h>
-#define TFTYPE	void
-#define TFRET()
+#define TFTYPE	unsigned int __stdcall
+#define TFRET() return 0
 #define THANDLE	HANDLE
 #else
 #include <pthread.h>
--- rtmpdump/librtmp/log.h.orig	2014-03-02 18:20:23.000000000 +0000
+++ rtmpdump/librtmp/log.h	2014-03-12 09:23:45.011220200 +0000
@@ -49,9 +49,9 @@
 void RTMP_LogSetCallback(RTMP_LogCallback *cb);
 void RTMP_LogSetOutput(FILE *file);
 #ifdef __GNUC__
-void RTMP_LogPrintf(const char *format, ...) __attribute__ ((__format__ (__printf__, 1, 2)));
-void RTMP_LogStatus(const char *format, ...) __attribute__ ((__format__ (__printf__, 1, 2)));
-void RTMP_Log(int level, const char *format, ...) __attribute__ ((__format__ (__printf__, 2, 3)));
+void RTMP_LogPrintf(const char *format, ...) __attribute__ ((__format__ (gnu_printf, 1, 2)));
+void RTMP_LogStatus(const char *format, ...) __attribute__ ((__format__ (gnu_printf, 1, 2)));
+void RTMP_Log(int level, const char *format, ...) __attribute__ ((__format__ (gnu_printf, 2, 3)));
 #else
 void RTMP_LogPrintf(const char *format, ...);
 void RTMP_LogStatus(const char *format, ...);
--- rtmpdump/librtmp/handshake.h.orig	2014-03-02 18:20:23.000000000 +0000
+++ rtmpdump/librtmp/handshake.h	2014-03-12 09:26:06.474683800 +0000
@@ -134,6 +134,7 @@
   HMAC_setup(ctx, secretKey, 128);
   HMAC_crunch(ctx, pubKeyOut, 128);
   HMAC_finish(ctx, digest, digestLen);
+  (void) digestLen;
 
   RTMP_Log(RTMP_LOGDEBUG, "RC4 In Key: ");
   RTMP_LogHex(RTMP_LOGDEBUG, digest, 16);
--- rtmpdump/librtmp/rtmp.c.orig	2014-03-02 18:20:23.000000000 +0000
+++ rtmpdump/librtmp/rtmp.c	2014-03-12 09:27:08.312536200 +0000
@@ -272,7 +272,7 @@
       return NULL;
   }
 #elif defined(USE_GNUTLS) && !defined(NO_SSL)
-  gnutls_certificate_allocate_credentials((gnutls_certificate_credentials*) &ctx);
+  gnutls_certificate_allocate_credentials((gnutls_certificate_credentials_t *) &ctx);
   if (gnutls_certificate_set_x509_key_file(ctx, cert, key, GNUTLS_X509_FMT_PEM) != 0) {
     gnutls_certificate_free_credentials(ctx);
     return NULL;
--- rtmpdump/librtmp/rtmp.c.orig	2014-03-12 09:27:27.855517800 +0000
+++ rtmpdump/librtmp/rtmp.c	2014-03-12 09:32:34.378941400 +0000
@@ -2485,7 +2485,7 @@
 #elif defined(USE_GNUTLS)
 typedef struct md5_ctx	MD5_CTX;
 #define MD5_Init(ctx)	md5_init(ctx)
-#define MD5_Update(ctx,data,len)	md5_update(ctx,len,data)
+#define MD5_Update(ctx,data,len)	md5_update(ctx,(unsigned) len,(const uint8_t *) data)
 #define MD5_Final(dig,ctx)	md5_digest(ctx,MD5_DIGEST_LENGTH,dig)
 #else
 #endif
--- rtmpdump/librtmp/rtmp.c.orig	2014-03-12 09:33:00.164715700 +0000
+++ rtmpdump/librtmp/rtmp.c	2014-03-12 09:35:01.513625100 +0000
@@ -3572,7 +3572,6 @@
   uint8_t hbuf[RTMP_MAX_HEADER_SIZE] = { 0 };
   char *header = (char *)hbuf;
   int nSize, hSize, nToRead, nChunk;
-  int didAlloc = FALSE;
   int extendedTimestamp;
 
   RTMP_Log(RTMP_LOGDEBUG2, "%s: fd=%d", __FUNCTION__, r->m_sb.sb_socket);
@@ -3700,7 +3699,6 @@
 	  RTMP_Log(RTMP_LOGDEBUG, "%s, failed to allocate packet", __FUNCTION__);
 	  return FALSE;
 	}
-      didAlloc = TRUE;
       packet->m_headerType = (hbuf[0] & 0xc0) >> 6;
     }
 
--- rtmpdump/librtmp/hashswf.c.orig	2014-03-02 18:20:23.000000000 +0000
+++ rtmpdump/librtmp/hashswf.c	2014-03-12 09:37:14.957070300 +0000
@@ -48,7 +48,7 @@
 #endif
 #undef HMAC_CTX
 #define HMAC_CTX	struct hmac_sha256_ctx
-#define HMAC_setup(ctx, key, len)	hmac_sha256_set_key(&ctx, len, key)
+#define HMAC_setup(ctx, key, len)	hmac_sha256_set_key(&ctx, len, (const uint8_t *) key)
 #define HMAC_crunch(ctx, buf, len)	hmac_sha256_update(&ctx, len, buf)
 #define HMAC_finish(ctx, dig, dlen)	dlen = SHA256_DIGEST_LENGTH; hmac_sha256_digest(&ctx, SHA256_DIGEST_LENGTH, dig)
 #define HMAC_close(ctx)
--- rtmpdump/librtmp/Makefile.orig	2014-03-12 09:49:51.274110300 +0000
+++ rtmpdump/librtmp/Makefile	2014-03-12 09:50:07.689194800 +0000
@@ -114,7 +114,7 @@
 	cp librtmp.pc $(LIBDIR)/pkgconfig
 	cp librtmp.3 $(MANDIR)/man3
 
-install_so:	librtmp$(SO_EXT)
+install_so:	librtmp$(SO_EXT) install_base
 	cp librtmp$(SO_EXT) $(SODIR)
 	$(INSTALL_IMPLIB)
 	cd $(SODIR); ln -sf librtmp$(SO_EXT) librtmp.$(SOX)
--- rtmpdump/Makefile.orig	2015-04-10 09:39:49.033600000 +0300
+++ rtmpdump/Makefile	2015-04-10 09:42:55.350600000 +0300
@@ -25,11 +25,9 @@
 LDFLAGS=-Wall $(XLDFLAGS)
 
 bindir=$(prefix)/bin
-sbindir=$(prefix)/sbin
 mandir=$(prefix)/share/man
 
 BINDIR=$(DESTDIR)$(bindir)
-SBINDIR=$(DESTDIR)$(sbindir)
 MANDIR=$(DESTDIR)$(mandir)
 
 LIBS_posix=
@@ -59,9 +57,9 @@
 $(PROGS): $(LIBRTMP)
 
 install:	$(PROGS)
-	-mkdir -p $(BINDIR) $(SBINDIR) $(MANDIR)/man1 $(MANDIR)/man8
+	-mkdir -p $(BINDIR) $(MANDIR)/man1 $(MANDIR)/man8
 	cp rtmpdump$(EXT) $(BINDIR)
-	cp rtmpgw$(EXT) rtmpsrv$(EXT) rtmpsuck$(EXT) $(SBINDIR)
+	cp rtmpgw$(EXT) rtmpsrv$(EXT) rtmpsuck$(EXT) $(BINDIR)
 	cp rtmpdump.1 $(MANDIR)/man1
 	cp rtmpgw.8 $(MANDIR)/man8
 	@cd librtmp; $(MAKE) install
--- rtmpdump/librtmp/Makefile.orig	2015-04-10 09:42:13.667400000 +0300
+++ rtmpdump/librtmp/Makefile	2015-04-10 09:51:38.129000000 +0300
@@ -117,5 +117,4 @@
 install_so:	librtmp$(SO_EXT) install_base
 	cp librtmp$(SO_EXT) $(SODIR)
 	$(INSTALL_IMPLIB)
-	cd $(SODIR); ln -sf librtmp$(SO_EXT) librtmp.$(SOX)
 
