diff -ur libopenmpt-0.2.8190-orig/Makefile libopenmpt-0.2.8190/Makefile
--- libopenmpt-0.2.8190-orig/Makefile	2017-05-26 13:28:48.878919800 +0800
+++ libopenmpt-0.2.8190/Makefile	2017-05-26 13:51:10.717699800 +0800
@@ -190,7 +190,7 @@
 
 # host setup
 
-ifeq ($(OS),Windows_NT)
+ifeq ($(OS),dummy)
 
 HOST=windows
 HOST_FLAVOUR=
@@ -224,6 +224,7 @@
 ifeq ($(UNAME_S),FreeBSD)
 HOST_FLAVOUR=FREEBSD
 endif
+LIBS_DEPS := -lrpcrt4
 
 endif
 
@@ -851,7 +852,7 @@
 	$(VERYSILENT)echo 'Version: $(DIST_LIBOPENMPT_VERSION)' >> $@.tmp
 	$(VERYSILENT)echo 'Requires.private: $(PC_REQUIRES_ZLIB) $(PC_REQUIRES_MPG123) $(PC_REQUIRES_OGG) $(PC_REQUIRES_VORBIS) $(PC_REQUIRES_VORBISFILE)' >> $@.tmp
 	$(VERYSILENT)echo 'Libs: -L$${libdir} -lopenmpt' >> $@.tmp
-	$(VERYSILENT)echo 'Libs.private: $(PC_LIBS_DL) $(PC_LIBS_LTDL)' >> $@.tmp
+	$(VERYSILENT)echo 'Libs.private: $(LIBS_DEPS) $(PC_LIBS_DL) $(PC_LIBS_LTDL)' >> $@.tmp
 	$(VERYSILENT)echo 'Cflags: -I$${includedir}' >> $@.tmp
 	$(VERYSILENT)mv $@.tmp $@
 
