diff --git a/Makefile.am b/Makefile.am
index 6ccaa99..4feded6 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,10 +1,10 @@
 AUTOMAKE_OPTIONS = foreign
 
-SUBDIRS = schroedinger doc tools testsuite
+SUBDIRS = schroedinger
 
 DISTCHECK_CONFIGURE_FLAGS=--enable-gtk-doc
 
-DIST_SUBDIRS = schroedinger doc tools testsuite
+DIST_SUBDIRS = schroedinger
 
 EXTRA_DIST = COPYING COPYING.GPL COPYING.LGPL COPYING.MIT COPYING.MPL \
 	     autogen.sh schroedinger.spec TODO NEWS orc.mak
diff --git a/configure.ac b/configure.ac
index 8caa560..074330c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -131,7 +131,6 @@ GTK_DOC_CHECK([1.0])
 AS_COMPILER_FLAG(-Wall, SCHRO_CFLAGS="$SCHRO_CFLAGS -Wall")
 if test "x$SCHRO_CVS" = "xyes"
 then
-  AS_COMPILER_FLAG(-Werror, SCHRO_CFLAGS="$SCHRO_CFLAGS -Werror")
   AS_COMPILER_FLAG(-Wno-strict-overflow, SCHRO_CFLAGS="$SCHRO_CFLAGS -Wno-strict-overflow")
 fi
 
@@ -229,23 +228,9 @@ AC_SUBST(pkgconfigdir)
 
 AC_CONFIG_FILES([
 Makefile 
-doc/Makefile
 schroedinger/Makefile
 schroedinger/cuda/Makefile
 schroedinger/opengl/Makefile
-testsuite/Makefile
-testsuite/cache/Makefile
-testsuite/cuda/Makefile
-testsuite/perf/Makefile
-testsuite/coder/Makefile
-testsuite/dirac/Makefile
-testsuite/installation/Makefile
-testsuite/manual/Makefile
-testsuite/opengl/Makefile
-testsuite/prototype/Makefile
-testsuite/streams/Makefile
-testsuite/tables2/Makefile
-tools/Makefile
 schroedinger/schroversion.h
 schroedinger-uninstalled.pc
 schroedinger.pc
